### PR TITLE
Refactor: HorizontalMediaView

### PR DESCRIPTION
**Pull request by phillipm**
- Break-up method into smaller helpers
- Remove navigation icon code, which is never executed because `int navStyle` is always set to `NAVIGATION_NONE`

perhaps easiest reviewed commit-by-commit

### CODE REVIEW

File: app/src/org/commcare/views/HorizontalMediaView.java [hunk 1]
```diff
@@ -27,137 +26,112 @@
...
-    }
-
-    private HorizontalMediaView(Context c, EvaluationContext ec) {
         super(c);
-        mTextView = null;
-        mAudioButton = null;
-        mImageView = null;
-        this.ec = ec;
-        this.iconDimension = (int)getResources().getDimension(R.dimen.menu_icon_size);
+        this.iconDimension = (int) getResources().getDimension(R.dimen.menu_icon_size);
```
Comment (amstone326) [inline app/src/org/commcare/views/HorizontalMediaView.java]: Thought our official style policy on casting was no space, but can't remember where we actually have that stuff documented so not positive..

Comment (phillipm) [inline app/src/org/commcare/views/HorizontalMediaView.java]: haha, it is!
There was a brief point where I changed android studio versions and forgot to install our code format config file.

Comment (amstone326) [inline app/src/org/commcare/views/HorizontalMediaView.java]: Ok wanna change that back really quick and then I'll merge?

**Comment by phillipm** [comment]
updated

**Pull request merged**